### PR TITLE
fix/table layout, add search placeholder

### PR DESCRIPTION
**Pull request by mugnivenko**
https://app.clickup.com/t/1pqg5ae

### CODE REVIEW

**Comment by mugnivenko** [comment]
https://app.clickup.com/t/1pqg5ae

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
Make Approvals table:
Screenshot 2021-11-02 at 17 05 47
like Broker Assistance table:
Screenshot 2021-11-02 at 17 05 47
e.g. look at the 'Josephine Tester' column, in one it fits the other it doesn't?

File: components/Pages/CarePackages/ReviewPackageDetails/index.js [hunk 1]
```diff
@@ -137,8 +136,6 @@ const ReviewPackageDetails = ({
     }));
   };
 
-  const isActivePackage = packageStatus < 6;
```
Comment (akeamwoodsND) [inline components/Pages/CarePackages/ReviewPackageDetails/index.js:140]: Why remove this?

File: pages/care-package/[guid]/details.js [hunk 2]
```diff
@@ -174,14 +174,16 @@ const PackageDetailsPage = () => {
       className="package-details"
       showEditActions
       openedPopup={openedPopup}
-      buttons={[
+      buttons={![
```
Comment (akeamwoodsND) [inline pages/care-package/[guid]/details.js:177]: This is overly complicated logic

File: components/Pages/CarePackages/ReviewPackageDetails/index.js (hunk 1 above)
Comment (mugnivenko) [reply components/Pages/CarePackages/ReviewPackageDetails/index.js]: i removed this change, it's not relevant to this branch

File: pages/care-package/[guid]/details.js (hunk 2 above)
Comment (mugnivenko) [reply pages/care-package/[guid]/details.js]: changes were removed

**Review by akeamwoodsND — APPROVED** [review]

**Pull request merged**